lib/expand/join.go

```diff
@@ -830,7 +830,12 @@ func (p *Peer) tryConnect(operationID string) (ctx *operationContext, err error)
 			return nil, utils.Abort(err)
 		}
 		if trace.IsCompareFailed(err) {
-			p.printStep("Waiting for another operation to finish at %v", addr)
+			p.WithError(err).Info("Failed to create expand operation.")
+			if utils.IsClusterDegradedError(err) {
+				p.printStep("Waiting to cluster to become healthy")
```

Comment:
for cluster

Also, maybe elaborate a little bit, like "Cluster is currently degraded, waiting for it to become healthy"